docs/docs/dev-manuals/services/media.mdx

```diff
@@ -3,18 +3,23 @@ import CustomImage from "@site/src/components/CustomImage";
 # Media Service
 
 The media service provides the bounded context `Media`.
-It is responsible for managing medias.
+It is responsible for managing media files.
 The media services uses a Nginx reverse proxy to make the fixed domain of MinIO pre-signed URLs configurable through environment variables.
 
+A pre-signed URL is used as a reference to a specific media. It is generated for each `Media` entity when the `path` attribute is requested.
+This is necessary for access control to specific media files. A `Media` entity cannot be queried by a user client directly.
+It is used in within the service mesh by other services to reference medias in the services datatypes.
+Pre-signed URLs link media to authenticated service datatypes, chaining authentication. Each request of the `path` attribute on `Media` generates a new pre-signed URL with a limited validity duration.
```

Comment:
> So, in other words the presigned URLS are generated automatically without my interference?
And an attachment will always vanish once the presigned URL is no longer valid?

Users reading this, probably

Reply:
> > So, in other words the presigned URLS are generated automatically without my interference?
> > And an attachment will always vanish once the presigned URL is no longer valid?
> 
> Users reading this, probably

Yes, exactly